qiskit_addon_sqd/fermion.py

```diff
@@ -459,6 +469,18 @@ def enlarge_batch_from_transitions(
     return np.array(bitstring_matrix_augmented)
 
 
+def _antisymmetric_matrix_from_upper_tri(k_flat: np.ndarray) -> Array:
+    """Create an anti-symmetric matrix given the upper triangle."""
+    num_rows = int((1 + np.sqrt(1 + 8 * len(k_flat))) // 2)  # quadratic formula
```

Comment:
Here we can avoid floating point arithmetic by passing `norb` as an argument to this function.